appveyor.yml

```diff
@@ -13,7 +13,7 @@ skip_tags: true
 #---------------------------------#
 
 # Operating system (build VM template)
-os: Windows Server 2012
+os: Visual Studio 2015
```

Comment:
No, this also works. See for [example](https://github.com/aspnet/Mvc/blob/dev/appveyor.yml)